src/features/Consumer/components/StatDisplay.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Group,
+  Paper,
+  Text,
+  ThemeIcon,
+  SimpleGrid,
+  Container,
+  Box,
+} from "@mantine/core";
+import { IconArrowUpRight, IconArrowDownRight } from "@tabler/icons-react";
+
+const data = [
+  { title: "Weighted No of Stars", value: "4.0 Stars", diff: -20 },
+  { title: "No. of Real Reviews", value: "420", diff: -69 },
+];
+
+export const StatDisplay: React.FC = () => {
+  const stats = data.map((stat) => {
+    const DiffIcon = stat.diff > 0 ? IconArrowUpRight : IconArrowDownRight;
+
+    return (
+      <Paper
+        withBorder
+        p="xl"
+        radius="md"
+        key={stat.title}
+        style={{ height: "100%" }}
```

Comment:
You can do
```suggestion
        h="100%"
```
But i suspect this is redundant and you can completely remove it